src/modules/reporting/components/reporting-dispute-no-rep-state/reporting-dispute-no-rep-state.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import PropTypes from 'prop-types'
+
+// noinspection JSFileReferences
```

Comment:
Is this specific to IntelliJ?  

Ideally we exclude env specific comments w/in the CB.